docs/configure-zapi.md

```diff
@@ -189,3 +189,76 @@ exporters, the following parameters can be defined:
 * `instances_keys` (list): display names of labels to export with each data-point
 * `instance_labels` (list): display names of labels to export as a separate data-point
 * `include_all_labels` (bool): export all labels with each data-point (overrides previous two parameters)
+
+### Filter
+
+This guide provides detailed instructions on how to use the `filter` feature in ZapiPerf.
```

Comment:
We should say something about why you would want to use filter. Maybe something like this:

This guide provides instructions on how to use the `filter` feature in ZapiPerf. Filtering is useful when you want to query a subset of instances. Say you have a small number of high-value volumes that you want Harvest to collect performance metrics from every five seconds. You can't collect all volumes, every five seconds, because that would be too expensive. Filtering lets you create a separate template that includes only the high-value volumes.